lib/chef/knife/core/windows_bootstrap_context.rb

```diff
@@ -161,10 +161,27 @@ def latest_current_windows_chef_version_query
         end
 
         def win_wget
+          # I tried my best to figure out how to properly url decode and switch / to \
+          # but this is VBScript - so I don't really care that badly.
           win_wget = <<-WGET
 url = WScript.Arguments.Named("url")
 path = WScript.Arguments.Named("path")
 proxy = null
+If InStr(url, "file://") = 1 Then
```

Comment:
Take the above comment and put in in the VBS here with something like this, as up above the context is a little confusing, especially when vbscript just looks like random text to you. ;)
```
'* Handle file:// paths, but doesn't handle converting forward slashes to backslashes.
```